### PR TITLE
[FIX] : Imrpoving Acessibility

### LINKED ISSUE #48: Bug: Improving acessibility of the website
### Description

Currently, as you can see the accessibility score is pretty low, the issue is to make this score as high as possible.

- Some image elements do not have an alt tag
- Some image elements have alt tags but the description is very much different than what it should be
- The Links do not have a discernible name

### Proposed Solution

- You can generate the alt tags using [alt-text generator](https://alttext.in/)
- You can add `aria-label` or research more about how to make links accessible

### Screenshots

image

### Browser 🥦

Brave

### Checklist ✅

- [X] I checked and didn't find similar [issue](https://github.com/hack4bengal/s3/issues)
- [ ] I am willing to work on this issue (blank for no).

### PR DESCRIPTION
**Pull request by harith-hacky03**
All alt tags are changed to the names given by the alt generator. 
Links are improved to give better accessibility.

closes #48

### CODE REVIEW

File: src/components/private/faq/FAQ.jsx [hunk 1]
```diff
@@ -73,10 +73,11 @@ const Faq = () => {
                   href="https://discord.com/invite/hack4bengal-2-0-920970750154899476"
                   target="_blank"
                   rel="noopener noreferrer"
+                  
                 >
-                  https://discord.com/invite/hack4bengal-2-0-920970750154899476
+                  Join our Discord community for support in #🎫ticket-support
```
Comment (tamalCodes) [inline src/components/private/faq/FAQ.jsx]:
```suggestion
                  in #🎫ticket-support
```

File: src/components/shared/footer/Footer.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const Footer = () => {
               </a>
             ))}
           </div>
-          <img src={h4b} alt="logo" />
+          <img src={h4b} alt="a black and white logo" />
```
Comment (tamalCodes) [inline src/components/shared/footer/Footer.jsx]:
```suggestion
          <img src={h4b} alt="A black and white logo representing Hack4Bengal" />
```

File: src/components/shared/footer/Footer.jsx [hunk 3]
```diff
@@ -44,8 +44,9 @@ const Footer = () => {
               rel="noopener noreferrer"
               className="footer__content__bottom__mail"
             >
-              {footerContent.mail.address}
+            Contact us: {footerContent.mail.address}
```
Comment (tamalCodes) [inline src/components/shared/footer/Footer.jsx]: I guess we can remove this as the previous line says `Email Us for more details`

**Review by tamalCodes — CHANGES_REQUESTED** [review]
I think the score can be improved more, it shows errors for links and images too. [Check out the report.](https://pagespeed.web.dev/analysis/https-s3-website-elzd1i1hz-h4b-2-vercel-app/im5kwudcy9?form_factor=desktop)

image

**Comment by harith-hacky03** [comment]
@tamalCodes  Is it okay now?

**Comment by tamalCodes** [comment]
> @tamalCodes Is it okay now?

Paste the latest deployed link in [this tester](https://pagespeed.web.dev/) and see the score. If there are any errors, try and fix it.

**Comment by harith-hacky03** [comment]
ps

**Comment by harith-hacky03** [comment]
@tamalCodes with my experience if you want higher performance, accessibility and seo. The app needs to be in next js. It offers prefetching server side rendering and other advanced features. Thank you

**Comment by tamalCodes** [comment]
Hi there @harith-hacky03 

Changing tech stacks is not a solution in my opinion either. There is no data to be prefetched. All the data you see are essentially static. Serving static resources doesn't need a much-advanced tech stack I guess.


IF you visit [this link](https://pagespeed.web.dev/analysis/https-s3-website-op0qkda8i-h4b-2-vercel-app/llezp39hjn?form_factor=desktop) you will see that it says 

- `Image elements do not have [alt] attributes` meaning the alt tags are missing in some cases.
- `Lists do not contain only <li> elements and script supporting elements (<script> and <template>).` meaning we have used bad elements inside the `ul`
- Some elements might not be keyboard focusable/missing proper aria-roles


A11y has nothing to do with the tech stack, it has more to do with the number of missing things that we have and should have been there. Anyway Thank you so much for the contribution if you think you are done with the PR, let me know I will merge it.

**Pull request closed without merging**